addons/website_sale/controllers/main.py

```diff
@@ -230,10 +230,9 @@ def shop(self, page=0, category=None, search='', ppg=False, **post):
         if attrib_list:
             post['attrib'] = attrib_list
 
-        Product = request.env['product.template'].with_context(bin_size=True)
+        search_categories = Product = request.env['product.template'].with_context(bin_size=True)
```

Comment:
Why are you putting to `search_categories` the `bin_size=True` context?

Reply:
I guess it was just to save a LOC, but you're right; let me change that.